deployment.rst

```diff
@@ -167,6 +167,16 @@ as you normally do:
     using :ref:`Symfony Flex <symfony-flex>`) before running this command so
     that the ``post-install-cmd`` scripts run in the ``prod`` environment.
 
+.. tip::
+
+    To improve performance and if you have Symfony Flex 1.2 or later installed, you may
+    leverage a (feature)[https://symfony.com/doc/4.4/configuration.html#configuring-environment-variables-in-production]
```

Comment:
instead of embedding a hardcoded URL here we should make use of the `ref` role like this:

```rst
:ref:`feature <configuration-env-var-in-prod>`
```

Reply:
of course, I will improve this